packages/recoil-devtools/src/utils/getStyle.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * Recoil DevTools browser extension.
+ *
+ * @emails oncall+recoil
+ * @flow strict-local
+ * @format
+ */
+'use strict';
+
+export const getStyle = (
```

Comment:
Really appreciate this! I've delaying it because at some point I thought we would add some css-in-react framework but this makes things much simpler 👍